appendix-basic65.tex

```diff
@@ -5751,43 +5894,39 @@ \subsection{LINE INPUT\#}
 % ****
 % LIST
 % ****
-\label{basic65-list}
+\label{BASIC 65 Commands!LIST}
 \newpage
 \subsection{LIST}
 \index{BASIC 65 Commands!LIST}
 \begin{description}[leftmargin=2cm,style=nextline]
 \item [Token:] \$9B
 \item [Format:] {\bf LIST} [{\bf P}] [line range]
+\item [Usage:] Lists a range of lines from the BASIC program in memory.
 
-\item [Usage:] Used to list a range of lines from the BASIC program.
+               Given a single line number, {\bf LIST} lists that line.
 
-               {\bf line range} consists of the first and/or last
-               line to list, or a single line number.
-               If the first number is omitted, the
-               first BASIC line is assumed.
-               If the second number is omitted, the last BASIC line
-               is assumed.
+               Given a range of line numbers, {\bf LIST} lists all lines in
+               that range. A range can be two numbers separated by a hyphen
+               (\screentext{-}), a starting number and a hyphen, or a hyphen
+               and an ending number.
 
 \item [Format:] {\bf LIST} [{\bf P}] filename [{\bf,U} unit]
-
-\item [Usage:] Used to list a BASIC program directly from {\bf unit},
-               which by default is 8.
+\item [Usage:] Lists a range of lines from a BASIC program directly from a file.
 
 \item [Remarks:]
-
                 The optional parameter {\bf P} enables page mode.
                 After listing 24 lines, the listing will stop and display
```

Comment:
I tried this in 80x50 text mode of the latest beta rom and it will list 49 lines and pause there.

I'm 50/50 on whether we elaborate on this case though, maybe it's not necessary.

Reply:
Changed to "After listing a screenful of lines." :)